wd_helper.au3

```diff
@@ -1560,7 +1568,25 @@ Func _WD_DownloadFile($sURL, $sDest, $iOptions = Default)
 			FileWrite($hFile, $sData)
 			FileClose($hFile)
 
-			$bResult = True
+			$hWaitTimer = TimerInit()
+			; make sure that file is not used after download, for example by AV software scaning procedure
+			While 1
+				__WD_Sleep(100)
+				If @error Then
+					$iErr = @error
+					ExitLoop
+				ElseIf Not _WinAPI_FileInUse($sDest) Then
+					If @error Then
+						$iErr = $_WD_ERROR_FileIssue
+					Else
+						$bResult = True
+					EndIf
+					ExitLoop
+				ElseIf TimerDiff($hWaitTimer) > $_WD_DefaultTimeout Then
+					$iErr = $_WD_ERROR_Timeout
```

Comment:
I'm wondering if this should be `$_WD_ERROR_FileIssue` instead of `$_WD_ERROR_Timeout` since the download was successful and the "timeout" is related to a file issue.

What do you think?

Reply:
What about your recent statement about:
`Using a different error code for one of these might help with future troubleshooting efforts.`

Reply:
The current implementation works, but I don't think `$_WD_DefaultTimeout` is the best way to describe the issue being reported. One solution would be to use `@extended` as a way to differentiate one instance of `$_WD_ERROR_FileIssue` from the other.

If you don't like that, then we can just leave it as-is.